app/assets/stylesheets/views/_homepage.scss

```diff
@@ -11,31 +11,60 @@ body.homepage {
   }
 }
 
-#homepage-promo-banner {
```

Comment:
Oh wait, no, you're right. This is probably reintroducing recently removed stuff. I'll take another look.